nvflare/private/fed/server/server_state.py

```diff
@@ -40,14 +40,14 @@ class ServerState(ABC):
     ABORT_CURRENT_RUN = {ACTION: ABORT_RUN, MESSAGE: "Abort current run"}
     IN_SERVICE = {ACTION: SERVICE, MESSAGE: "Server in service"}
 
+    logger = logging.getLogger("FederatedServer")
```

Comment:
should it be
``logger = logging.getLogger("ServerState")``

Reply:
fixed.